boot/arch/ia64/_link.ld.in

```diff
@@ -6,6 +6,7 @@ SECTIONS {
 		loader_start = .;
 		*(BOOTSTRAP);
 		*(.text);
+		*(.text.*);
```

Comment:
I wonder which symbols we missed this way :-)

Reply:
Without this, we couldn't enable `-ffunction-sections` here.